Readme.md

```diff
@@ -9,9 +9,12 @@ Like Game Maker, ENIGMA comprises both an easy to use Drag & Drop system as well
 Although EDL adopts GML's very lax syntax in its scripting, the inputted code is actually parsed and translated into valid C++. This compilation improves size and performance of the language by incredible amounts, while almost magically maintaining its simplicity of use.
 
 ## Install
-In order to keep the ENIGMA repository small and fast binary files are excluded. To install you will need to obtain the binary files for your chosen platform. Please read the `building.md` file for more information.
+Please see http://enigma-dev.org/download.htm for installation instructions.
```

Comment:
Not much reason to delete the justification for not including binaries.

Reply:
Yeah but that's the whole point of GitHub? This is common knowledge. I was concerned because we do have binaries under the releases page, of the old releases plus the ENIGMA Portable which I've been calling r5 informally. I'm afraid your wording there is just confusing for GitHub people.

Reply:
I suppose. This was very relevant information when we had just switched off of SVN, but nowadays, people are used to more procedure when they set out to use the contents of a repository.